src/applications/vaos/utils/appointment.jsx

```diff
@@ -17,7 +17,15 @@ export function isCommunityCare(appt) {
 }
 
 export function isVideoVisit(appt) {
-  return !!appt.vvsAppointments;
+  return !!appt.vvsAppointments || appt.appointmentKind === 'MOBILE_GFE';
+}
+
+export function isGFEVideoVisit(appt) {
+  return appt.appointmentKind === 'MOBILE_GFE';
```

Comment:
I think this `appointmentKind` property is inside the `vvsAppointments` items, based on the mock data we have.